src/lib.rs

```diff
@@ -4,6 +4,7 @@ use ciborium::de::from_reader;
 use ic_cdk::api::management_canister::main::{
     CanisterId, CanisterIdRecord, CanisterInstallMode, CreateCanisterArgument, InstallCodeArgument,
 };
+use ic_cdk::api::management_canister::provisional::CanisterSettings;
```

Comment:
You should rather use `ic_cdk::api::management_canister::main::CanisterSettings` and add it to the list above. The `provisional` module is just a re-export.

Reply:
Done